frontend/src/components/TeamRegistration.js

```diff
@@ -150,21 +151,56 @@ const TeamRegistration = () => {
       </div>
 
       <div class="mx-auto max-w-screen-md">
-        <div class="mt-12">
-          <h4 class="text-lg font-bold text-blue-500">Register your team(s)</h4>
+        <div class="mt-4">
+          <h4 class="mb-4 text-lg font-bold text-blue-500">Registered teams</h4>
```

Comment:
here, can we add count of teams registered so far ?. 
Like "Registered teams (12)"